bindings/python/unicorn/unicorn_py3/unicorn.py

```diff
@@ -1227,11 +1359,17 @@ def _do_reg_read_batch(self, reglist, vallist, count) -> int:
 
         return uclib.uc_context_reg_read_batch(self._context, reglist, vallist, count)
 
+    def _do_reg_write_batch(self, reglist, vallist, count) -> int:
```

Comment:
Duplicate definition of line 762?

Reply:
They belong to two different classes (engine and context). The underlying methods are using the same names to allow seamless OOP.

Reply:
Okay I see